spec/tic_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require '../lib/tic'
+require '../lib/players'
+def preprocess(indexes)
+  i = 0
+  while i < 2
+    if indexes[i] < 1 or indexes[i] > 3
+      puts 'invalid move only in range of 1-3'
+      return [1, 1]
+    end
+    indexes[i] = indexes[i] - 1 if indexes[i] == 1
+    indexes[i] = indexes[i] + 1 if indexes[i] == 3
+    i += 1
+  end
+  indexes
+end
+
+def gameobj(player)
+  tic = Tic.new
```

Comment:
- [ ] use `let` to declare variables when testing. Change this line and any other to `let(:tic){Tic.new}`